src/Form/Type/FormatterType.php

```diff
@@ -57,9 +57,9 @@ class FormatterType extends AbstractType
     private $templateManager;
 
     /**
-     * @param ConfigManagerInterface|null   $configManager   An Ivory CKEditor bundle configuration manager
-     * @param PluginManagerInterface|null   $pluginManager   An Ivory CKEditor bundle plugin manager
-     * @param TemplateManagerInterface|null $templateManager An Ivory CKEditor bundle template manager
+     * @param ConfigManagerInterface|null   $configManager   A FOS CKEditor bundle configuration manager
+     * @param PluginManagerInterface|null   $pluginManager   A FOS CKEditor bundle plugin manager
+     * @param TemplateManagerInterface|null $templateManager A FOS CKEditor bundle template manager
```

Comment:
Please allow FOS or Ivory here

Reply:
Remove typehint and add an `if` in method?